src/parcsr_ls/ams.c

```diff
@@ -409,12 +409,12 @@ hypreCUDAKernel_ParCSRMatrixFixZeroRows( HYPRE_Int      nrows,
 
    for (HYPRE_Int j = p1 + lane; j < q1; j += HYPRE_WARP_SIZE)
    {
-      l1_norm += fabs(A_diag_data[j]);
+      l1_norm += hypre_cabs(A_diag_data[j]);
```

Comment:
`hypre_cabs` is a host function, one cannot use it in kernels. I understand you want it to work with complex, but we need to something else, if we want to support it.